src/config/absinthe.ts

```diff
@@ -18,5 +18,19 @@ const socketConnection = new PhoenixSocket.Socket(SOCKET, {
   },
 });
 
+// we should try to reconnect ws connection only finite (5) times and then abort and prevent
+// unnecessary load on the server
+// watch for websocket error event using onError
+const WEB_SOCKET_RETRY_ATTEMPTS = 5;
```

Comment:
I think this should be in constants file?

Reply:
I was thinking about it too, not sure why I ruled against it :) Let me make the change.